src/main/webapp/WEB-INF/views/drivers/getAllDrivers.jsp

```diff
@@ -0,0 +1,26 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All Drivers</title>
+</head>
+<body>
+<h1>All drivers:</h1>
+<table>
+  <tr>
+    <td>ID</td>
+    <td>NAME</td>
+    <td>LICENSE_NUMBER</td>
+    <td>DELETED</td>
+  </tr>
+  <c:forEach items="${drivers}" var="driver">
+    <tr>
+      <td><c:out value="${driver.id}" /></td>
```

Comment:
Nice, approved.
Minor remark, in some places you use obj.attribute, in other places obj.getAttribute().